src/main/java/seedu/ravi/Ravi.java

```diff
@@ -24,9 +26,9 @@
      */
     public static void main(String[] args) {
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
-            logger.getLogger().info("Shutdown hook - Saving all data...");
-            InputOutputManager.save();
-            InputOutputManager.saveNusMods();
+            logger.getLogger().info("Shutdown hook");
+            System.out.println();
+            TextUi.outputToUser(MESSAGE_GOODBYE);
```

Comment:
I realise now that if one were to enter `bye` into the prompt, it will cause `MESSAGE_GOODBYE` to be printed twice, once in the execution of the `ExitCommand` and once here. This is because Java cannot differentiate between a normal `System.exit` and CTRL-C.

Reply:
We can just remove the printing of `MESSAGE_GOODBYE` in `ExitCommand` because whenever the shutdown occurs it will print the message anyways

Reply:
Changed the message to be printed in shutdown hook